scanamo/src/main/scala/org/scanamo/DynamoFormat.scala

```diff
@@ -89,6 +89,11 @@ trait DynamoFormat[T] {
   def read(av: DynamoValue): Either[DynamoReadError, T]
   def read(av: AttributeValue): Either[DynamoReadError, T] = read(DynamoValue.fromAttributeValue(av))
   def write(t: T): DynamoValue
+
+  def iso[U](r: T => U, w: U => T): DynamoFormat[U] = DynamoFormat.iso(r, w)(this)
+  def xmap[U](r: T => Either[DynamoReadError, U])(w: U => T): DynamoFormat[U] = DynamoFormat.xmap(r)(w)(this)
```

Comment:
Let's be consistent and use a single parameter block here too.

Reply:
... and if you dont mind applying the same in the companion, i.e. a single parameter block for both `xmap` and `coercedXmap`, that would be 🙏

Reply:
Did it in f680771cf355f2c54934a8b0fbb7bb815f040527